src/Lib/Server/Auth.hs

```diff
@@ -84,13 +84,13 @@ isLoggedInHandler token = timedAction "isLoggedInHandler" $ do
   unless isLoggedIn $ throwError (NotAllowed "Revoked Session")
   return NoContent
 
-logoutHandler :: (MonadSession m, MonadLogger m) => Text -> m NoContent
+logoutHandler :: (MonadSession m, KatipContext m) => Text -> m NoContent
 logoutHandler token = timedAction "logoutHandler" $ do
   mPayload <- decodeAndVerifyJWTToken token
   case mPayload of
     Just JWTPayload{..} -> do
       deleteSession jwtUserId
       return NoContent
     Nothing -> do
-      $(logDebug) $ token <> " was used to logout when it was invalid"
+      $(logTM) DebugS $ ls $ token <> " was used to logout when it was invalid"
```

Comment:
Could `$(logTM) DebugS` go into separate function as it's used often?

Reply:
Hmm... That would be great... I would like to as well. But I don't see any way to do this 😞 `logTM` generates function which takes `DebugS`. And we always need to write `$(logTM)` in order to get actual source code position. If I move this to separate function then it will always have the same source code position.